build/template-install-dependencies.yaml

```diff
@@ -22,7 +22,6 @@ steps:
   displayName: 'Use .Net Core SDK 8'
   inputs:
     version: 8.x  
-    includePreviewVersions: true
   condition: eq(variables['TargetNet8'], 'True')

```

Comment:
this needs to be removed as well.

Reply:
`includePreviewVersions: true` was removed unless you're talking about a different line?

Reply:
yes line 25 or 26 - the one with `condition: ...`

Reply:
done :)